projects/rero/ng-core/src/lib/search-input/search-input.component.ts

```diff
@@ -14,20 +14,27 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-import { Component, Input, Output, EventEmitter } from '@angular/core';
+import { Component, Input, Output, EventEmitter, ViewChild } from '@angular/core';
 
 @Component({
   selector: 'ng-core-search-input',
   templateUrl: './search-input.component.html'
 })
 export class SearchInputComponent {
-  @Input()
-  displayLabel = true;
-
+  /** The current search input object from the template. */
+  @ViewChild('searchinput', { static: false }) input: any;
+  @Input() displayLabel = true;
   @Input() placeholder = 'search';
   @Output() search = new EventEmitter<string>();
   @Input() searchText = '';
```

Comment:
Could you just group @Input, @Output and @ViewChild properties and add a space between them? It's easier to read the code.